dbt/models/eia923/core_eia923__monthly_generation_fuel_nuclear/schema.yml

```diff
@@ -13,6 +13,9 @@ sources:
           - name: nuclear_unit_id
           - name: energy_source_code
           - name: fuel_type_code_pudl
+            data_tests:
+              - dbt_expectations.expect_column_values_to_be_in_set:
+                  value_set: ["nuclear"]
```

Comment:
Simpler than testing that 100% of the generation in this table is from nuclear (though that is also true!)